### PR TITLE
Line enable/disable

### LINKED ISSUE #229: Add enable/disable line endpoints
As the documentation is almost live, the code can be added here as well.

### PR DESCRIPTION
**Pull request by Artamus**
Should wait until the endpoints and documentation are fully live before merging.

Resolve #229

### CODE REVIEW

File: src/services/LineService.ts [hunk 1]
```diff
@@ -93,6 +93,50 @@ export default class LineService {
...
+   *
+   * Calls the following HTTP API: `POST /lines/<ID>/enable`
+   *
+   * For example:
+   *
+   * ```javascript
+   * await Qminder.lines.enable(1425);
+   * ```
+   * @param line the Line or the ID of the line to be enabled.
+   * @returns A Promise that resolves when the line was deleted, and rejects
```
Comment (boxmein) [inline src/services/LineService.ts]: Typo here? "Line was deleted"

File: src/services/LineService.ts [hunk 2]
```diff
@@ -93,6 +93,50 @@ export default class LineService {
...
+   *
+   * Calls the following HTTP API: `POST /lines/<ID>/disable`
+   *
+   * For example:
+   *
+   * ```javascript
+   * await Qminder.lines.disable(1425);
+   * ```
+   * @param line the Line or the ID of the line to be disabled.
+   * @returns A Promise that resolves when the line was deleted, and rejects
```
Comment (boxmein) [inline src/services/LineService.ts]: Also a typo here? "Line was deleted"

File: src/services/LineService.ts (hunk 1 above)
Comment (Artamus) [reply src/services/LineService.ts]: Typos indeed

**Comment by Artamus** [comment]
There are no tests yet, which needs to be resolved as well.

**Pull request merged**